firestore-bigquery-export/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Version 0.1.14
+
+feature - added warsaw (europe-central2) location (#677)
+
+fixed - Updated file naming for bq imports (#589)
```

Comment:
The file naming PR was only a change to the npm script for firestore-bigquery-export, not the extension itself, so please remove this line. More context [here](https://github.com/firebase/extensions/pull/650#issuecomment-828536860)

Reply:
Done.